pkg/db/db_releases.go

```diff
@@ -134,6 +134,28 @@ func (h *DBHandler) DBSelectReleaseByVersion(ctx context.Context, tx *sql.Tx, ap
 	return h.processReleaseRow(ctx, err, rows, ignorePrepublishes, true)
 }
 
+func (h *DBHandler) DBSelectReleaseByReleaseNumbers(ctx context.Context, tx *sql.Tx, app string, releaseVersion types.ReleaseNumbers, ignorePrepublishes bool) (*DBReleaseWithMetaData, error) {
+	span, ctx := tracer.StartSpanFromContext(ctx, "DBSelectReleaseByVersion")
+	defer span.Finish()
+	selectQuery := h.AdaptQuery(`
+		SELECT created, appName, metadata, manifests, releaseVersion, environments, revision
+		FROM releases  
+		WHERE appName=? AND releaseVersion=?  AND revision= ?
+		LIMIT 1;
+	`)
+	span.SetTag("query", selectQuery)
```

Comment:
Please replace all `SetTag("query"...` with this:
```suggestion
	span.SetTag("query", selectQuery)
	MarkSpanAsDB(span, selectQuery)
```